src/main/java/edu/greenblitz/tobyDetermined/Robot.java

```diff
@@ -1,30 +1,27 @@
 package edu.greenblitz.tobyDetermined;
 
 
-import edu.greenblitz.tobyDetermined.commands.BatteryDisabler;
 import edu.greenblitz.tobyDetermined.subsystems.Battery;
 import edu.greenblitz.tobyDetermined.subsystems.Dashboard;
-import edu.greenblitz.tobyDetermined.subsystems.Limelight;
 import edu.greenblitz.tobyDetermined.subsystems.swerve.SwerveChassis;
 import edu.wpi.first.net.PortForwarder;
 import edu.wpi.first.wpilibj.TimedRobot;
-import edu.wpi.first.wpilibj.livewindow.LiveWindow;
 import edu.wpi.first.wpilibj2.command.CommandScheduler;
 
 public class Robot extends TimedRobot {
 	@Override
 	public void robotInit() {
 		CommandScheduler.getInstance().enable();
 		Dashboard.init();
-		Limelight.getInstance();
-		initPortForwarding();
-		LiveWindow.disableAllTelemetry();
-		Battery.getInstance().setDefaultCommand(new BatteryDisabler());
+//		Limelight.getInstance();
```

Comment:
Commented code

Reply:
uncommented

Reply:
Didn't mean to just uncomment code, check if needed or not